mobilestyle.css

```diff
@@ -168,7 +168,14 @@
 
 .menuPageList li a:hover {
   color: #36b37f;
-  transition: all 0.3s;
+  transition: all 0.6s;
+}
+
+@media (min-width: 375px) and (max-width: 768px) {
+  .menuPageList li a:hover {
+    color: #36b37f;
```

Comment:
- [ ] Great job so far guys, there are just a few changes you need to make and you'll be on your way to the next milestone :wink: 
- [ ] As mentioned in the Project Requirements, the hamburger button and menu need to follow the design from [your chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).
- Currently your `menu items` don't have the same font color as seen in the Figma design. So please fix this :100: 

| Your Design | The Figma |
|--------|--------|
| ![image](https://github.com/GeekyHacks/My-Personal-Portfolio/assets/102699958/ee656c3e-a1fb-4d4c-a51f-4ffd8d949461) | ![image](https://github.com/GeekyHacks/My-Personal-Portfolio/assets/102699958/946c9477-3e14-4309-84d2-9f3c447ac07a) |